config/routes.rb

```diff
@@ -2,5 +2,13 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
![image](https://github.com/Ouail-01/Blog-App/assets/93039359/7a125e32-3d5d-458e-8e51-b0d65449a639)

- [ ] It would be nice if you fix the link highlighted in the image above because it is broken right now 😨 Kindly fix it or remove it to make your readme descriptive and professional 👍